cpp/tests/test_random_number_generator.cpp

```diff
@@ -4,35 +4,43 @@
 #include <algorithm>
 #include <iterator>
 
-TEST(RandomNumberGenerator, blocks)
```

Comment:
There is a test on the subsequence_counter functioning in persons. Should we test the helper functions in `cpp/memilio/utils/random_number_generator.h` more? I can't see the code coverage right now, as there are merge conflicts. Maybe check after resolving.

Reply:
added more tests for the counter based RNG stuff